homeassistant/components/ecobee/sensor.py

```diff
@@ -108,7 +109,11 @@ def device_class(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        if self._state in [ECOBEE_STATE_CALIBRATING, ECOBEE_STATE_UNKNOWN, "unknown"]:
+        if self._state in [
+            ECOBEE_STATE_CALIBRATING,
+            ECOBEE_STATE_UNKNOWN,
+            STATE_UNKNOWN,
```

Comment:
If `self._state` holds an ecobee api state string we should not use a home assistant constant to compare it.

Reply:
Maybe `ECOBEE_STATE_BARE_UNKNOWN`?